week2-section1/exercise3/main.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+Exercise 3 - Shapes and Areas
+Create two classes: a Circle class and a Square class. For each class, take in the necessary information to calculate its area and perimeter.� Remember to validate input.
+
+Ask the user if they want to create a circle or a square. For their pick, ask the necessary information to create the shape. Then, tell them the area and the perimeter of their shape.
+*/
+
+#include <iostream>
+#include <cmath>
+#include <iomanip>
+
+class Circle
+{
+private:
+
+	float radius;
+
+public:
+	Circle()
+	{
+		radius = 0.0f;
+	}
+	Circle(float InRadius)
+	{
+		radius = InRadius;
+	}
+	void CalculatePerimeter()
```

Comment:
These functions should be Getters. Please follow the exercise instructions carefully.